flink-runtime/src/test/java/org/apache/flink/runtime/rest/RestClientTest.java

```diff
@@ -106,15 +103,15 @@ public void testInvalidVersionRejection() throws Exception {
                             EmptyRequestBody.getInstance(),
                             Collections.emptyList(),
                             RuntimeRestAPIVersion.V0);
-            Assert.fail("The request should have been rejected due to a version mismatch.");
+            fail("Expected exception not thrown.");
         } catch (IllegalArgumentException e) {
```

Comment:
```suggestions
  assertThatThrownBy(() ->
                    restClient.sendRequest(
                            unroutableIp,
                            80,
                            new TestMessageHeaders(),
                            EmptyMessageParameters.getInstance(),
                            EmptyRequestBody.getInstance(),
                            Collections.emptyList(),
                            RuntimeRestAPIVersion.V0))
                    .isInstanceOf(IllegalArgumentException.class);
```

By the way, would you mind to change all this pattern of expecting an exception?

Reply:
thx, i have fixed it